src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
```

Comment:
The `div` tag is a non-semantic element. It's better to use semantic elements when possible. In this case, the `section` tag might be a better choice as it represents a standalone section of the document, which seems to be the role of this `div`. The same applies to the other `div` elements in your code.